setup.cfg

```diff
@@ -19,27 +19,27 @@ include_package_data = True
 package_dir =
     = src
 packages = find_namespace:
-python_requires = >=3.8,!=3.9.7,<3.11
+python_requires = >=3.9,!=3.9.7,<3.11
 
 install_requires =
-    orquestra-quantum==0.9.0
-    orquestra-qiskit==0.9.0
-    orquestra-cirq==0.8.0
+    orquestra-quantum~=0.9.0
+    orquestra-qiskit~=0.9.0
+    # temporary hack until newest version of orquestra-cirq is released
+    orquestra-cirq @ git+https://github.com/zapatacomputing/orquestra-cirq.git
     networkx>=2.8.7
     # Used to define and run Orquestra workflows
     orquestra-sdk[all]==0.45.1
     matplotlib>=3.6
     numpy>=1.20
     more-itertools~=9.1.0
     pandas==1.5.3
-    # pyLIQTR was forked to remove dependencies on cirq-rigetti which has conflicting
-    # dependencies with ray (used by orquestra-sdk)
-    pyLIQTR @ git+https://github.com/SebastianMorawiec/pyLIQTR_test.git@fa3a0e410ff1cf34d6d060118268a319ac793604
+    pyLIQTR~=0.3.0
```

Comment:
Same here.

Reply:
Wait, isn't it being tagged to a version good enough?